### PR TITLE
feat: Added support for Django32

### PR DESCRIPTION
**Pull request by iamsobanjaved**
- Upgrading to Django 3.2.
- Upgraded codebase for python 3.8.
- Dropped support for Django 2.2 and 3.0
- Shifted CI to Github actions

Ran CI on GHA here: https://github.com/iamsobanjaved/xblock-google-drive/pull/1

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,12 +2,11 @@ language: python
...
 
 python:
-  - 3.5
-  - 3.8
+  - '3.8'
 env:
-  - TOXENV=django22
   - TOXENV=quality
-
+  - TOXENV=django31
```
Comment (awais786) [inline .travis.yml:8]: you are not using it.

**Review by awais786 — APPROVED** [review]

**Comment by awais786** [comment]
@natabene @jmbowman PR is ready but some one with permission can enable the github actions and merge it.

**Comment by natabene** [comment]
@awais786 Can you merge? If not, @jmbowman, do you know if it is expected behavior that Arbisoft folks don't have merge rights here?

**Comment by jmbowman** [comment]
@sarina or @feanil - could I get a review on the write permissions for this repo?  I think it's currently scoped narrower than most other openedx org repos because it came from edx-solutions.  I have merge permissions but the Arbi-BOM team members don't, and I don't want to be the blocker on PRs here unless there's an explicit decision to keep write access very restricted.

**Comment by feanil** [comment]
@jmbowman makes sense to me, I've given the `@openedx/arbi-bom` team write access to this repo. I've also gone ahead and reviewed the rest of the edx-solutions source repos and ensured that they all also have write access for the arbi-bom team.

**Comment by natabene** [comment]
@awais786 You should be goo to merge, I guess.

**Comment by awais786** [comment]
@feanil tests are not running.

**Comment by feanil** [comment]
@awais786 it looks like the tests in this repo were never migrated from travis to github actions.  Can someone on your team port them over?

**Comment by feanil** [comment]
@awais786 they won't run as a part of the security requirements of the repo, since the test file doesn't already exist on master

**Pull request merged**